public/app/core/utils/explore.ts

```diff
@@ -448,7 +448,7 @@ export const getFirstQueryErrorWithoutRefId = (errors: DataQueryError[]) => {
     return null;
   }
 
-  return errors.filter(error => (error.refId ? false : true))[0];
+  return errors.filter(error => (error && error.refId ? false : true))[0];
```

Comment:
Btw I seen this multiple times but is there a reason to write like this instead of:
```
!!(error && error.refId)
```

Reply:
I use it for clarity only. `!!` can make it hard to read, but that just my personal preference.